### PR TITLE
[Merged by Bors] - chore: silence tests, and enforce this

### PR DESCRIPTION
**Pull request by kim-em**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: test/basicTactics.lean [hunk 1]
```diff
@@ -4,6 +4,7 @@ example : ∀ a b : Nat, a = b → b = a := by
   introv h
   exact h.symm
 
+#check Nat
```
Comment (eric-wieser) [inline test/basicTactics.lean:7]: I'm guessing that the enforcement is currently broken; perhaps you need to capture stderr instead?

Comment (semorrison) [reply test/basicTactics.lean:7]: Yes, it's strange --- `make test` fails locally! I'll push a minor change (better file names that won't be clobbered by parallel tests) and try again.

Comment (semorrison) [reply test/basicTactics.lean:7]: Successful failure now!

File: test/eqns.lean [hunk 2]
```diff
@@ -27,7 +27,12 @@ will not have the desired effect.
...
+
+/--
+warning: declaration uses 'sorry'
+-/
+#guard_msgs in
 -- the above should error as the above equation would not have changed the output of the below
 example (n : Nat) : t = n := by
   rw [t]
-  sorry
+  admit
```
Comment (kmill) [inline test/eqns.lean:38]: I didn't know `admit` lets you avoid the `sorry` linter... Interesting.

Other tests define a `test_sorry` axiom and apply it.

**Review by kmill — COMMENTED** [review]
bors r+

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**